runtime/doc/lsp.txt

```diff
@@ -1318,65 +1317,58 @@ Lua module: vim.lsp.client                                        *lsp-client*
                                `start()` take precedence over the global
                                command registry. Each key must be a unique
                                command name, and the value is a function which
                                is called if any LSP action (code action, code
-                               lenses, ...) triggers the command.
+                               lenses, …) triggers the command.
+      • {detached}?            (`boolean`, default: true) Daemonize the server
+                               process so that it runs in a separate process
+                               group from Nvim. Nvim will shutdown the process
+                               on exit, but if Nvim fails to exit cleanly this
+                               could leave behind orphaned server processes.
+      • {flags}?               (`table`) A table with flags for the client.
+                               The current (experimental) flags are:
+                               • {allow_incremental_sync}? (`boolean`,
+                                 default: `true`) Allow using incremental sync
+                                 for buffer edits
+                               • {debounce_text_changes} (`integer`, default:
+                                 `150`) Debounce `didChange` notifications to
+                                 the server by the given number in
+                                 milliseconds. No debounce occurs if `nil`.
+                               • {exit_timeout} (`integer|false`, default:
+                                 `false`) Milliseconds to wait for server to
+                                 exit cleanly after sending the "shutdown"
+                                 request before sending kill -15. If set to
+                                 false, nvim exits immediately after sending
+                                 the "shutdown" request to the server.
+      • {get_language_id}?     (`fun(bufnr: integer, filetype: string): string`)
+                               Language ID as string. Defaults to the buffer
+                               filetype.
+      • {handlers}?            (`table<string,function>`) Map of LSP method
+                               names to |lsp-handler|s.
       • {init_options}?        (`lsp.LSPObject`) Values to pass in the
                                initialization request as
                                `initializationOptions`. See `initialize` in
                                the LSP spec.
-      • {name}?                (`string`, default: client-id) Name in log
-                               messages.
-      • {get_language_id}?     (`fun(bufnr: integer, filetype: string): string`)
-                               Language ID as string. Defaults to the buffer
-                               filetype.
+      • {name}?                (`string`) (default: client-id) Name in logs
```

Comment:
This has lost of special default formatting. Should be `(`string`, default: client-id)`

Reply:
What is the markup for that here: https://github.com/neovim/neovim/blob/43f3c4a48f6cc3a4c4173591a59b8e3fcdb96759/runtime/lua/vim/lsp/client.lua#L92-L93 ?

Reply:
thanks, fixed in https://github.com/neovim/neovim/pull/33687